src/highlight/HighlightAnnotations.tsx

```diff
@@ -39,7 +40,7 @@ export default class HighlightAnnotations extends React.PureComponent<Props> {
                 <HighlightList annotations={sortedAnnotations} onSelect={this.handleAnnotationActive} />
 
                 {/* Layer 3: Drawn (unsaved) incomplete annotation target, if any */}
-                {isCreating && <HighlightCreator className="ba-HighlightAnnotations-creator" />}
+                <HighlightCreator className="ba-HighlightAnnotations-creator" pageEl={pageEl} />
```

Comment:
It seems like we will eventually need two components, `HighlightCreator` when we explicitly enter highlight mode and handles clicking and dragging out the highlight, and then `HighlightPromoter` which will watch for selection events and present that popup to create a highlight out of the text selection.